devel/patchelf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                patchelf
+version             0.10.0
+categories          devel
+platforms           darwin
+license             GPL-3
+maintainers         @lkollar
+description         Modify dynamic ELF executables
+long_description    PatchELF is a simple utility for modifying existing ELF executables and libraries
+homepage            https://github.com/NixOS/patchelf
+
+github.setup        NixOS patchelf 0.10
+
+checksums           sha256  717c7b21a8a480611f3b52f595f7529ace60adc2d2bfde89a0558cb6ce9d659e \
+                    rmd160  9aa4451b699f61ded89b2a8c582ec1100598787c \
+                    size    99623
+
+use_autoreconf      yes
+autoreconf.cmd      ./bootstrap.sh
+
+depends_build       port:autoconf \
+                    port:automake
+
+configure.args      --disable-dependency-tracking \
+                    --disable-silent-rules
```

Comment:
it looks like the build system adds `-std=c++11` suggesting that you'll need to tell MacPorts to make sure to get a compatible compiler, with `compiler.cxx_standard   2011`